components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -750,11 +750,25 @@ base::PostTaskAndReplyWithResult(file_task_runner_.get(), FROM_HERE,
                      AsWeakPtr()));
 }
 
-void RewardsServiceImpl::RestorePublishers() {
-  if (!Connected())
+void RewardsServiceImpl::OnRestorePublishersUI(const int32_t result) {
+  if (static_cast<ledger::Result>(result) != ledger::Result::LEDGER_OK) {
     return;
+  }
 
-  bat_ledger_->RestorePublishers();
+  for (auto& observer : observers_) {
+    observer.OnExcludedSitesChanged(
+      this, "-1", ledger::PUBLISHER_EXCLUDE::ALL);
+  }
+}
+
+void RewardsServiceImpl::RestorePublishersUI() {
```

Comment:
The naming convention used here is confusing as I am reading this in that we are restoring (or updating) the publishers UI, however speaking with @NejcZdovc this function is called from the UI.